lib/jobs.rb

```diff
@@ -1,6 +1,7 @@
 module Jobs
   SIMPLE_SEARCHES = '(job|employment|internship)s?'
-  JOB_RELATED_KEYWORDS = '((position|opening|posting|job|employment|intern(ship)?|seasonal|trabajo|puesto|empleo|vacante)s?|(opportunit|vacanc)(y|ies))|(posicion|ocupacion|oportunidad|federal)es|gobierno'
+  JOB_RELATED_KEYWORDS = '((position|opening|posting|job|employment|intern(ship)?|seasonal|trabajo|puesto|empleo|vacante)s?|(opportunit|vacanc)(y|ies))|(posicion|ocupacion|oportunidad|federal)(es\b|\b)|gobierno'
```

Comment:
Instead of:
`(posicion|ocupacion|oportunidad|federal)(es\b|\b)`

you should just need:
`(posicion|ocupacion|oportunidad|federal)(es)?`

```ruby
[17] Usasearch(main)> /(posicion|ocupacion|oportunidad|federal)(es)?/ === 'posicion'
true
[18] Usasearch(main)> /(posicion|ocupacion|oportunidad|federal)(es)?/ === 'posiciones'
true
```